blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,413 @@
+    width: 17px;
+  }
+
+  .icon-fb,
+  .icon-twitter,
+  .icon-yt,
+  .icon-instagram,
+  .icon-linkedin {
+    display: flex;
+  }
+
+  
+  .v2-footer .v2-footer__title {
+    font-size: 14px;
+    font-weight: 400;
+    margin-bottom: 5px;
+    line-height: 22px;
+  }
+  
+  .v2-footer ul li a,
+  .v2-footer .v2-footer-bottom-section li a {
+    color: var(--c-secondary-silver);
+  }
+  
+  .v2-footer ul li a:hover {
+    color: var(--c-secondary-white);
+    text-decoration: none;
+  }
+  
+  .v2-footer .v2-footer__title ~ ul {
+    display: flex;
+    flex-flow: row wrap;
+  }
+
+  .v2-footer .v2-footer-bottom-section li {
+    padding: 10px 0;
+  }
+  
+  .v2-footer .v2-footer__title ul li {
+    width: 50%;
+  }
+  
+  .v2-footer .v2-footer-bottom-section-content {
+    display: flex;
+    align-items: start;
+    font-size: 12px;
+    width: 91%;
+    flex-wrap: wrap;
+    gap: 20px;
+  }
+  
+  .v2-footer .v2-footer-bottom-section li a:hover {
+    color: var(--c-primary-white);
+  }
+  
+  .v2-footer .footer-form-and-social-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .v2-footer .v2-footer-bottom-section {
+    margin-bottom: 38px;
+    border-top: 1px solid var(--c-tertiary-cool-gray);
+    width: 100%;
+  }
+
+  .v2-footer .v2-footer-links-section {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+
+  .v2-footer .v2-footer-links-section ul {
+    min-width: 50%;
+  }
+  
+  .v2-footer .v2-footer-form-header .v2-footer__title {
+    font-family: var(--ff-body);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 1.92px;
+    text-transform: uppercase;
+    margin-bottom: 12px;
+  }
+
+  .v2-footer .v2-footer-main-content {
+    display: grid; 
+    grid-template-columns: 1fr; 
+    grid-template-rows: 1fr 1fr 1fr 1fr; 
+    gap: 0 0; 
+    /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+    grid-template-areas: 
+      "v2-logo"
+      "v2-content"
+      "v2-form"
+      "v2-social"; 
+      height: 72vh;
+  }
+  
+  .v2-social { 
+    grid-area: v2-social;
+    margin-top: 0;
+  }
+
+  .v2-content { grid-area: v2-content; }
+
+  .v2-form { 
+    grid-area: v2-form;
+    height: 15vh;
+  }
+
+  .v2-logo { 
+    grid-area: v2-logo; 
+    height: 15vh;
+  }
+
+  .v2-footer .v2-footer-links-section ul li {
+    margin-bottom: 14px;
+  }
+
+  /* desktop   */
+  @media (min-width: 1200px) {
+    .v2-footer {
+      padding-top: 96px;
+    }
+  
+    .v2-footer .v2-footer-content {
+      display: flex;
+      flex-flow: row wrap;
+    }
+  
+    .v2-footer .v2-footer-links-section {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-start;
+      gap: 20px;
+      flex-wrap: nowrap;
+    }
+
+    .v2-footer .v2-footer-links-section ul {
+      min-width: 0;
+      width:160px;
+    }
+
+    .v2-footer .v2-footer-links-section ul li{
```

Comment:
It has the same css rule at line 160, this is redundant .
Also gap between each link is `16px` 
Refer here https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2243%3A159264&mode=dev

Reply:
Updated